base/util.go

```diff
@@ -757,6 +757,10 @@ func SafeSlice(data []byte, from int, to int) ([]byte, error) {
 	return data[from:to], nil
 }
 
+func StringPointer(s string) *string {
+	return &s
+}
+
```

Comment:
What's the rationale for a helper method here, instead of just using &s in the calling code?  Just for the cases where you're passing a literal?

Reply:
Originally I was using a literal and so  it was a convenience.  The aws-sdk uses that pattern for various types (for example here: https://github.com/aws/aws-sdk-go/blob/71a68668c287f9ad58d0c58c78e27f08eb39fa49/aws/convert_types.go#L6) and it's a nice convenience when you have to pass literals to structs that expect pointers.

Reply:
I'm changing it to just using `&` though and removing `StringPointer()` since I ended up using a variable rather than a literal.